app/views/certificates/primary_certificate/show.html.erb

```diff
@@ -3,6 +3,7 @@
 <%= render 'pages/certification/hero' %>
 <div class="ncce-programmes-activity">
   <div class="govuk-width-container">
+    <%= render ProgressBarComponent.new(current_user, @programme) %>
```

Comment:
This is a really great way to keep the main views simple by dropping in a re-usable component that you made. 👍

Reply:
Thank you